### DISCUSSION TITLE
Text prompt instance segmentation

### DISCUSSION THREAD

**Discussion by narges-tk** [opengeos/segment-geospatial#229] [Q&A] [answered]
Hi!
Using text prompt SAM, I get just semantic segmentation. Could you please guide me on how I can do instance segmentation based on text prompt SAM?
Thanks in advance!

**Comment by giswqs** [comment, marked as answer]
For now, samgeo only support a single class segmentation. For multi-class segmentation, you can try running the text prompt multiple times.
